### PR TITLE
feat(orb): take in account comment from CircleCI Partner Engineering …

### PR DESCRIPTION
**Pull request by jeremyds**
…team

### CODE REVIEW

File: src/@orb.yml [hunk 1]
```diff
@@ -22,33 +22,28 @@ examples:
...
+        type: string
+    docker:
+      - image: gitguardian/ggshield::<<parameters.tag>>
     environment:
       CIRCLE_RANGE: << parameters.base_revision >>...<<parameters.revision>>
-    executor: ggshield-executor
     steps:
       - checkout
-      - scan-command
+      - ggshield scan -v ci
```
Comment (EricRibeiro) [inline src/@orb.yml]: When invoking command-line programs you have to use run: https://circleci.com/docs/2.0/configuration-reference/#run.

The way it is right now probably won't work.

**Review by EricRibeiro — CHANGES_REQUESTED** [review]
Your `ggshield scan -v ci` command has to be part of a `run` step. Check https://circleci.com/docs/2.0/configuration-reference/#run for more details.

File: src/@orb.yml [hunk 2]
```diff
@@ -22,33 +22,28 @@ examples:
...
-    steps:
-      - run: ggshield scan ci
+
 jobs:
   scan:
     parameters:
       base_revision:
         description: pipeline base_revision
         type: string
+        default: ""
```
Comment (agateau-gg) [inline src/@orb.yml:35]: Any reason we have a default value for `base_revision` and not for `revision`?

File: src/@orb.yml [hunk 3]
```diff
@@ -22,33 +22,28 @@ examples:
...
-commands:
-  scan-command:
-    steps:
-      - run: ggshield scan ci
+
 jobs:
   scan:
     parameters:
       base_revision:
         description: pipeline base_revision
```
Comment (agateau-gg) [inline src/@orb.yml]: This description is not super helpful. What about something like this: "ID of the first commit to scan. Leave empty to only scan the latest commit."

File: src/@orb.yml [hunk 4]
```diff
@@ -22,33 +22,28 @@ examples:
...
         type: string
+        default: ""
       revision:
         description: pipeline revision
```
Comment (agateau-gg) [inline src/@orb.yml]: Same thing here, I suggest something like this: "ID of the last commit to scan. Leave empty to scan up to the latest commit".

File: src/@orb.yml [hunk 5]
```diff
@@ -22,33 +22,28 @@ examples:
           jobs:
             - ggshield/scan:
                 name: ggshield-scan # best practice is to name each orb job
-                base_revision: << pipeline.git.base_revision >>
+                base_revision: <<pipeline.git.base_revision>>
```
Comment (agateau-gg) [inline src/@orb.yml:25]: .circleci/config.yml uses spaces inside the brackets, this file does not. I don't have strong opinions on which one is best, but it should be the same in all files.

**Review by agateau-gg — CHANGES_REQUESTED** [review]

**Pull request merged**